### DISCUSSION TITLE
Add Romania to countries available in choropleth

### DISCUSSION THREAD

**Discussion by omniconvert** [apache/superset#27383] [General]
## Description

Romania as a country is missing from the choropleth maps.

**Comment by omniconvert** [comment]
Not sure if bug or feature request at this point :)

**Comment by michael-s-molina** [comment]
Feature request 😄 Converting to discussion.

**Comment by rusackas** [comment]
You can just find the `.ipynb` Juypyter notebook, run it using `jupyter notebook`, add Romania to the list, and run the workbook. It generates everything you need to open a PR (including the updated workbook).

**Reply by omniconvert** [reply]
Thanks @rusackas for the imbold. 
I forked the repo. I cloned it locally. I did a pip install jupyter and notebook and other dependancies. 
I managed to open the notebook on localhost. And I tried running all steps. 

But I always get errors. First they were related to the files from naturalearthdata. I downloaded the manually. 

Then I always get something else. Sometimes 'france' or 'italy' is not defined. 

Now I get 
```
---------------------------------------------------------------------------
NameError                                 Traceback (most recent call last)
Cell In[16], line 2
      1 france_copy = france.copy()
----> 2 reposition(france_copy, france.name=='Guadeloupe', 57.4, 25.4, 1.5, 1.5)
      3 reposition(france_copy, france.name=='Martinique', 58.4, 27.1, 1.5, 1.5)
      4 reposition(france_copy, france.name=='Guyane française', 52, 37.7, 0.35, 0.35)

NameError: name 'reposition' is not defined
```
